.readthedocs.yaml

```diff
@@ -12,7 +12,7 @@ sphinx:
 build:
   os: ubuntu-22.04
   tools:
-    python: "3"
+    python: "3.12"
```

Comment:
Should we also recommend pinning to the specific version in `starbase` ?

Reply:
https://github.com/canonical/starbase/blob/b5c869d85018cea23b1b7378678429f4303c5ed1/.readthedocs.yaml#L17-L20

Reply:
I vote yes.

Reply:
Seconded.

Reply:
https://github.com/canonical/starbase/issues/283